src/assets/styles/sakura.scss

```diff
@@ -1,25 +1,12 @@
-/* stylelint-disable */
-/* Sakura.css v1.4.1
- * ================
- * Minimal css theme.
- * Project: https://github.com/oxalorg/sakura/
- */
-/* Body */
+* {
```

Comment:
Let's keep this for now, but I suspect that having a global CSS style will cause problems in the future. 

Later on we'll have another component that needs different styling. If we edit this global style then all the other components that rely on it will be broken. If we create a new style with a more specific selector, then that will work but over time we'll have a complex hierarchy of CSS styles. 

This is part of why I wanted to try out [MUI](https://mui.com/). JB suggested that we try MUI because it makes styling simpler. But first I'd like to try the module way that we learned in class. And then on later Issues, we can try refactoring it to use MUI.

Reply:
Sorry for this, we'll try out MUI